THIRD-PARTY.md

```diff
@@ -60,6 +60,9 @@ SPDX-License-Identifier: BSD-3-Clause
   The test image in testsuite/cryptomatte/src/cryptoasset.exr is a small crop
   of a sample image from the Cryptomatte project.
 
+* WebP (c) Copyright 2010 Google Inc.
+  https://developers.google.com/speed/webp
+
```

Comment:
This isn't needed (not in THIRD-PARTY.md anyway) because no WebP code is vendored into our repository.

Reply:
Ah, of course.